models/action.go

```diff
@@ -192,6 +193,24 @@ func (a *Action) GetRepoLink() string {
 	return "/" + a.GetRepoPath()
 }
 
+// GetRepositoryFromMatch returns a *Repository from a username/repo string
+func GetRepositoryFromMatch(match string) (*Repository, error) {
+	parts := strings.Split(match, "/")
```

Comment:
You know you could have already had the regexp split this for you?

Reply:
ah yea, that would make more sense. Do you think its worth it to ditch the separate function for this and just call GetRepositoryByOwnerAndName  etc... directly then? This function isn't doing much I just wasn't sure if calling GetRepositoryByOwnerAndName and checking for errors three separate times in a row was too much code duplication.

Reply:
If I was to refactor I would look at whether you could reduce the amount of code duplication or split up updateissuecommit. That's a very long function and removing your getrepositorybymatch function would only make it longer.

Reply:
Agree, will keep it in for now to avoid this being even longer. Adjusted to have matching done in regex itself thx.